torch/onnx/symbolic_opset9.py

```diff
@@ -393,6 +393,9 @@ def squeeze(g, self, dim=None):
 
 
 def prelu(g, self, weight):
+    self_sizes = self.type().sizes()
```

Comment:
Please check self is completeTensor,  something like `self.isCompleteTensor()`. Otherwise, accessing sizes() is unsafe.

Reply:
Thanks! I have added it.